.gitattibutes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
`.gitattibutes` -> `.gitattributes`. However, if everything worked with `.gitattibutes`, this file is probably obsolete

Reply:
Afk. Can you push a commit deleting and then merge? 

Too many open prs and I think we need to get good ones in before more conflicts